src/components/components.js

```diff
@@ -538,20 +545,34 @@ function NullTooltip() {
 // Tooltip component that doesn't show tooltip for empty/undefined tooltip.
 export class TooltipComponent extends React.Component {
 	static contextType = ReactContext;
-	setOverlayRef = (elem) => {
-		this.overlayElem = elem;
+
+	constructor(props) {
+		super(props);
+		this.state = {show: null};
```

Comment:
I'll change that. Also, your upper comment about renaming packages sounds good. I'm just wondering does it mean that all the projects that are using the bs3 theme (like laji) should change the dependency to this format `"react-bootstrap-3": "npm:react-bootstrap@^0.32.22"` or how does it work?

Reply:
Good question... let's investigate that before making the changes.

Reply:
I tried to make an alias `"react-bootstrap-3": "npm:react-bootstrap@0.33.1",`. That doesn't work unfortuntaly, because the typings break. Making an alias `"@types/react-bootstrap-3": "npm:@types/react-bootstrap@^0.32.22"` didn't work, because the type package is self-importing and produces errors like this:

```
node_modules/@types/react-bootstrap-3/lib/Accordion.d.ts:2:23 - error TS2307: Cannot find module 'react-bootstrap' or its corresponding type declarations.

2 import { Sizes } from 'react-bootstrap';
```

Which is impossible to solve afaik, unless we just ditch the typings.

`react-bootstrap` v5 has the typings included in the package itself, so it doesn't have that problem.

Two options I can think of to solve this: 

1. Making laji-form a monorepo. Both themes could just use `react-bootstrap` and the different themes would have their own `package.json`s with different versions of `react-bootstrap`.
2. Don't rename `react-bootstrap` v3.

2nd option is much simpler, so maybe we should go with that. Maybe rename `react-bootstrap` as `react-bootstrap-5` anyway? BTW I tested on laji-repo, and it seems that the client of laji-form must take care of aliasing the package in it's `package.json`. Installing laji-form becomes a bit more difficult, but we could make it a bit easier with some documentation in the README.

I've thought about 1st option before - it would make installing laji-form simpler, as you wouldn't have to install the packages needed for the theme separately... Maybe it's still too much hassle as the themes aren't that much used part of laji-form.